docs/installation.md

```diff
@@ -60,6 +60,84 @@ The sources for `watchmaker` are available from the [GitHub repo][2].
+necessary Python packages.  Packages are available for Windows and
+Linux.
+
+1.  Download the standalone package for your desired platform.
+
+    *   Watchmaker standalone packages can be downloaded from a GitHub
+        Releases page or an AWS S3 bucket.
+    *   The [GitHub Releases][3] page shows the available
+        Watchmaker versions and includes links to the
+        Windows and Linux packages, and SHA256 hashes.
+    *   The [AWS S3 bucket][4] also contains versions of Watchmaker packages
+        and corresponding SHA256 hashes.
+    *   The latest version of Watchmaker,
+        for Windows and Linux, is available in a special [latest bucket][5].
+    *   On S3, the latest version of Watchmaker is available in a Windows
+        package called
+        `watchmaker-latest-standalone-windows-amd64.exe`. The
+        corresponding SHA256 hash is called
+        `watchmaker-latest-sha256-windows-amd64.json`.
+
+        From PowerShell, the Windows package can be downloaded
+        as follows:
+
+        ```powershell
+        PS C:\wam> $url = "https://s3.amazonaws.com/watchmaker/releases/latest/watchmaker-latest-standalone-windows-amd64.exe"
+        PS C:\wam> (New-Object System.Net.WebClient).DownloadFile($url, "watchmaker.exe")
+        ```
+
+    *   The Linux package is called
+        `watchmaker-latest-standalone-linux-x86_64`. The
+        corresponding SHA256 hash is called
+        `watchmaker-latest-sha256-linux-x86_64.json`.
+
+        From the command line, the Linux package can be downloaded
+        as follows:
+
+        ```shell
+        # curl -so watchmaker https://s3.amazonaws.com/watchmaker/releases/latest/watchmaker-latest-standalone-linux-x86_64
+        ```
+
+    *   For the latest package, the version of Watchmaker can be determined by
+        viewing the contents of the SHA256 hash file or by executing the
+        package with the `--version` flag.
+
+2.  Verify the integrity of the standalone package.
+
+    Compare the SHA256 hash contained in the downloaded hash file to
+    a hash you compute for the package.
+
+    For Linux, execute this command to compute the SHA256 hash:
+      
+    ```shell
+    # shasum -a 256 watchmaker-latest-standalone-linux-x86_64
+    ```
+
+    For Windows, execute this command to compute the SHA256 hash:
+
+    ```powershell
+    PS C:\wam> Get-FileHash watchmaker-latest-standalone-windows-amd64.exe | Format-List
+    ```
+
+3.  Set executable access permission.
+
+    For Linux, you will need to set the access permissions to allow the
+    standalone executable to run. Below is an example:
+    
+    ```shell
+    # chmod +x watchmaker-latest-standalone-linux-x86_64
+    ```
+
+
 [0]: https://pip.pypa.io/en/stable/
 [1]: https://python-guide.readthedocs.io/en/latest/starting/installation/
 [2]: https://github.com/plus3it/watchmaker
+[3]: https://github.com/plus3it/watchmaker/releases/
+[4]: https://s3.console.aws.amazon.com/s3/buckets/watchmaker/releases/
```

Comment:
These console links are not accessible by users... And S3 buckets don't have a "directory" view, so users won't be able to see the directory structure and files. We should probably just describe the URL format so they can construct versioned URLs if they want, and for the "latest" pkg just provide the full URLs since they don't change.